spacelift/data_bitbucket_datacenter_integration.go

```diff
@@ -113,7 +111,7 @@ func dataBitbucketDatacenterIntegrationRead(ctx context.Context, d *schema.Resou
 		} `graphql:"bitbucketDatacenterIntegration(id: $id)"`
 	}
 
-	variables := map[string]interface{}{"id": defaultBitbucketDatacenterIntegrationID}
+	variables := map[string]interface{}{"id": ""}
```

Comment:
do we check for empty string everywhere in the backend? if I remember correctly we forgot it somewhere but maybe not

Reply:
Yup, I double-checked that. For all integrations, we have: `if args.ID == nil || *args.ID == "" {`. I'll add a test for that in the backend to ensure we will be compatible -> [PR](https://github.com/spacelift-io/backend/pull/6468).